airbyte-integrations/connectors/source-opsgenie/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9.13-alpine3.15 as base
```

Comment:
Any reason why you upgraded the image from `3.9.11` better to keep all the connectors on the same image for build times and debugging purposes

Reply:
Reverted 👍

Reply:
Fixed